ONSdigital/takeon-ui
# Feature/spp1372 display first time clear

# Description
A new dashboard display has been created to show First Time Clears. There is also a foundation for adding in other dashboards as well. The new dashboard has been implemented through the current DC UI and therefore can be automatically deployed.
https://collaborate2.ons.gov.uk/jira/browse/SPP-1372

# How to test?
Testing has only been manual as we have not implemented any BDD.
To test use the spp1372 environment. You will need to set yourself up on cognito and fargate to access the UI.
Navigate to https://dc-ui-spp1372.es-take-on.aws.onsdigital.uk/display_dashboard. Visually check the page loads and the data is the same as in the DB, and that columns can be sorted.
We loaded the synth-anon data and modified it slightly by adding a column for firstValidationCount.

# Links
https://github.com/ONSdigital/takeon-core-infrastructure/pull/112
